addons/pos_hr/static/src/css/pos.css

```diff
@@ -130,3 +125,8 @@
 .pos .pos-rightheader .header-button.lock-button:hover {
     color: rgb(197, 52, 0);
 }
+
+.cashier-button:hover {
+    background: rgb(104,69,95);
+    cursor: pointer;
```

Comment:
there is already a pointer cursor, so no need, and you can move the "hover backgroud" in point of sale module